src/routes/package-edit.js

```diff
@@ -103,7 +103,7 @@ class PackageEditRoute extends Component {
       }
 
       if ('allowKbToAddTitles' in values) {
-        model.allowKbToAddTitles = values.allowKbToAddTitles;
+        model.allowKbToAddTitles = values.allowKbToAddTitles === 'true'; // turn string into boolean
```

Comment:
does `!!` not work?

Reply:
Nah, because JavaScript. The value coming from the radio is a string of either `"true"` or `"false"`. 
![image](https://user-images.githubusercontent.com/2072894/40741986-a6be66e2-6412-11e8-837b-cb07d502de55.png)

The best way to turn a string "boolean" to a real boolean is to triple equal compare it to string true haha.